website/docs/kubernetes/deploying-a-pod-to-kubernetes.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 10
+title: Deploying a pod
+description: Deploying a pod to Kubernetes 
+keywords: [podman desktop, podman, containers, pods, migrating, kubernetes]
+tags: [migrating-to-kubernetes]
+---
+
+# Deploying a pod to Kubernetes
+
+With Podman Desktop, you can deploy a pod to your Kubernetes cluster.
+
+#### Prerequisites
+
+* You registered the Kubernetes cluster in your kubeconfig file: *<your_kubernetes_cluster>*. For example, [Creating a kind cluster](kind/creating-a-kind-cluster.md).
+* Your pod, running or stopped, is available on the **Pods** page: *<your_pod>*.
+* The Kubernetes namespace to deploy to already exists.
+
+#### Procedure
+
+1. Click **Podman Desktop tray > Kubernetes > Context > *<your_kubernetes_cluster>*** to set your Kubernetes context.
+1. Open **Podman Desktop dashboard > Pods > *<your_pod>*** to see the **Pod Details** page.
+1. Click <icon icon="fa-solid fa-rocket" size="lg" /> to generate a Kubernetes pod.
```

Comment:
is it normal that i don't see the icon `<icon icon="fa-solid fa-rocket" size="lg" />` when i display as rich diff?

Reply:
Yes